### DISCUSSION TITLE
Combing scans with different number of range bins

### DISCUSSION THREAD

**Discussion by rwall2** [ARM-DOE/pyart#1312] [Q&A]
Hello. I have two pyart radar objects that are each approximately half of a full volume of scans from a single radar. The first half volume consists of 7 elevation scans (from 0.5 degrees to 6 degrees) with 1248 range bins. The second half volume consists of 6 elevation scans (from 7 degrees to 20 degrees) with 625 range bins. The spatial resolution is the same at 200 meters for each of the half volumes.

Is there a method within pyart to combine these radar objects?

Thanks.

**Comment by scollis** [comment]
What is the goal? If you want to map them to a cartesian grid (gridding) you can call the grid code with a tuple.  If you want a radial object we can probably put together a quick cook book.. adding _FillValue to the end to the second radars array I need to go back and check what join_radar does… (Max and others are likely to get to that before me)  From: rwall2 ***@***.***>Date: Thursday, November 3, 2022 at 8:47 AMTo: ARM-DOE/pyart ***@***.***>Cc: Subscribed ***@***.***>Subject: [ARM-DOE/pyart] Combing scans with different number of range bins (Discussion #1312)Hello. I have two pyart radar objects that are each approximately half of a full volume of scans from a single radar. The first half volume consists of 7 elevation scans (from 0.5 degrees to 6 degrees) with 1248 range bins. The second half volume consists of 6 elevation scans (from 7 degrees to 20 degrees) with 625 range bins. The spatial resolution is the same at 200 meters for each of the half volumes.Is there a method within pyart to combine these radar objects?Thanks.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

**Reply by rwall2** [reply]
I prefer the quickest method, as I'm trying to make displaying the radar scans operational. I would think extending the second half volume to have 1248 gates and filling the tailing gates with zeros would work. Is that what you meant by _FillValue?

**Comment by mgrover1** [comment]
@rwall2 do you have sample data for this use-case?

**Reply by rwall2** [reply]
I can't supply any data unfortunately because it is proprietary. However, Scott mentioned join_radar. The join_radar  method seems to have done the job. It appears in my case that join_radar assigns to the new (joined) radar the number of range gates of the first half of the radar volume (i.e. the larger number of gates). Thanks for your help!
